frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -0,0 +1,391 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  Switch,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  ActionGroup,
+  Button,
+} from '@patternfly/react-core';
+import {
+  CaretDownIcon,
+  LockIcon,
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+
+import {
+  createModalLauncher,
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+  ModalComponentProps,
+  ModalFooter,
+} from '@console/internal/components/factory/modal';
+import { apiVersionForModel } from '@console/kubevirt-plugin/integration-tests/tests/utils/selectors';
+import {
+  HandlePromiseProps,
+  withHandlePromise,
+} from '@console/internal/components/utils/promise-component';
+import { k8sCreate, k8sGet } from '@console/internal/module/k8s/resource';
+
+import { CephClusterKind, StoragePoolKind, OCSStorageClusterKind } from '../../../types';
+import { CephBlockPoolModel, OCSServiceModel } from '../../../models';
+import { CEPH_STORAGE_NAMESPACE, OCS_INTERNAL_CR_NAME } from '../../../constants/index';
+
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { referenceForModel } from '@console/internal/module/k8s';
```

Comment:
These 2 imports need to be moved up before the relative ones.

Reply:
ack